ui/desktop/package.json

```diff
@@ -36,7 +36,7 @@
     "start:desktop": "ember electron",
     "clean:coverage": "rm -rf coverage_*",
     "pretest": "yarn clean:coverage",
-    "test": "npm-run-all test:*",
+    "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
```

Comment:
Why did we add linting to this, I didn't notice it in admin UI but we should keep our linting and testing commands separate. Can you change this to just ember exam in both desktop and UI?

Reply:
@ZedLi absolutely we can